path_credentials.go

```diff
@@ -220,11 +229,20 @@ func (b *datastaxAstraBackend) pathCredentialsWrite(ctx context.Context, req *lo
 		"token":    token.Token,
 		"metadata": token.Metadata,
 	}
+
 	err = saveToken(ctx, token, req.Storage)
 	if err != nil {
 		return nil, err
 	}
 	resp := b.Secret(astraTokenType).Response(token.toResponseData(), internalData)
+
+	leaseTime, ok := d.GetOk("lease_time")
```

Comment:
If a customer does not want to add a lease time, we should support that. ( The token should not expire at all in that case)